react/features/base/media/actionTypes.js

```diff
@@ -9,6 +9,16 @@
 export const SET_AUDIO_MUTED = Symbol('SET_AUDIO_MUTED');
 
 /**
+ * The type of (redux) action to adjust the availability of the local audio.
+ *
+ * {
+ *     type: SET_AUDIO_AVAILABLE,
+ *     muted: boolean
+ * }
+ */
+export const SET_AUDIO_AVAILABLE = Symbol('SET_AUDIO_AVAILABLE');
```

Comment:
I know it wasn't ordered even before this PR, but we can order those alphabetically.